backend/api/Services/AssetDeckService.cs

```diff
@@ -11,10 +11,16 @@ public interface IAssetDeckService
 
         public abstract Task<AssetDeck?> ReadById(string id);
 
+        public abstract Task<AssetDeck?> ReadByAssetAndDeck(string asset, string deck);
+
         public abstract Task<AssetDeck> Create(CreateAssetDeckQuery newAssetDeck);
 
+        public abstract Task<AssetDeck> Create(CreateAssetDeckQuery newAssetDeck, Pose safePosition);
```

Comment:
```suggestion
        public abstract Task<AssetDeck> Create(CreateAssetDeckQuery newAssetDeck, List<Pose> safePositions);
```
If we should have this method, shouldn't it have a list as input?

Reply:
Agreed